### DISCUSSION TITLE
Room 4: BIC data

### DISCUSSION THREAD

**Discussion by morvenjrs** [IBAHCM/RPiR#276] [Archive]
Room 4 for practical series 4 are:

Leo Jhaveri - @LeoJha
Kat Hopkinson - @kitkatR
Morven Rae-Seaman - @morvenjrs

Repos can be found:
https://github.com/IBAHCM/LeoJhaBCI 
https://github.com/IBAHCM/kitkatrBCI 
https://github.com/IBAHCM/morvenjrsBCI

**Comment by kitkatr** [comment]
new: https://github.com/IBAHCM/kathopkinsonBCI.git
